src/test/resources/github310/invalid/data/MAGBST051782340_V1.TAB

```diff
@@ -0,0 +1,9600 @@
+   28402312.00,    43,   108,   432

```

Comment:
@qchaupds unless we actually need to do content validation, can we make sure we do not include these large data files in the repo? i would prefer these be empty files and we use the `--skip-content-validation` flag. that goes for all our tests for that matter. our repo will start to get massive with all these test data products if we aren't careful.